### ISSUE TITLE
Multi page print failing?

### ISSUE THREAD

**Issue by goedman** [subethaedit/SubEthaEdit#190]
Fill the following template, and post it on [GitHub Issues](https://github.com/subethaedit/subethaedit/issues/new) or send to <subethaedit-bugs@lone.monkey.productions>. Please note that the contents of the sent email can be shared on the Issue page. Please write the contents in English.

Title: 

-----------------------------------------------

## Short Description

Multi-page printing results in a single messed up line at the bottom on Big Sur.

## Steps to Reproduce the Issue

1. Describe

Edit a multi page file. Select print.

## Actual Result

Bottom line misses all formatting (new lines, new page, etc?)

## Environment

- SubEthaEdit: Version 5.1.5 (9716)
- System: Version 11.1 (Build 20C5061b)
- Language: English

```
Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Model Identifier: MacBookPro16,1
      Processor Name: 8-Core Intel Core i9
      Processor Speed: 2.3 GHz
      Number of Processors: 1
      Total Number of Cores: 8
      L2 Cache (per Core): 256 KB
      L3 Cache: 16 MB
      Hyper-Threading Technology: Enabled
      Memory: 32 GB
      System Firmware Version: 1554.60.15.0.0 (iBridge: 18.16.13026.5.2,0)
      Serial Number (system): C02ZX4JYMD6T
      Hardware UUID: 57FFA8AF-A330-5095-A67A-7C45CBCE5AA9
      Provisioning UDID: 57FFA8AF-A330-5095-A67A-7C45CBCE5AA9


```

[SEE_print_problem.pdf](https://github.com/subethaedit/SubEthaEdit/files/5648938/SEE_print_problem.pdf)

**Comment by monkeydom** [comment]
Thanks for the report. Fixed on develop with issue #180. 

If you use the website version you can use the beta build from issue https://github.com/subethaedit/SubEthaEdit/issues/181

which should include that fix. Sorry for the inconvenience.

**Comment by goedman** [comment]
Thank you, will close this issue!
